### PR TITLE
fix: wrong isResendable cond check in openChannel/useResendMessageCallback 

### PR DESCRIPTION
**Pull request by AhyoungRyu**
~Fixes https://sendbird.atlassian.net/browse/CLNP-905~

`message.isResendable` type is function, and its return type is boolean.  So the way we've checked the value like 
```
if (failedMessage?.isResendable) { ... }
```

has been wrong but it should be if we really want to check its existence. 
```
if (typeof failedMessage?.isResendable === 'boolean') { ... }
```

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
Wait, I need to check a bit more.

File: src/modules/OpenChannel/context/hooks/useResendMessageCallback.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ function useResendMessageCallback(
     logger.info('OpenChannel | useResendMessageCallback: Resending message has started', failedMessage);
     // eslint-disable-next-line no-param-reassign
     const { messageType, file } = failedMessage as FileMessage;
-    if (failedMessage && typeof failedMessage.isResendable === 'function' && failedMessage.isResendable) {
+    if (typeof failedMessage?.isResendable === 'boolean') {
```
Comment (liamcho) [inline src/modules/OpenChannel/context/hooks/useResendMessageCallback.ts:28]:
```suggestion
    if (typeof failedMessage?.isResendable === 'boolean' && failedMessage.isResendable) {
```
?

File: src/modules/Channel/context/hooks/useResendMessageCallback.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ function useResendMessageCallback({
   return useCallback((failedMessage) => {
     logger.info('Channel: Resending message has started', failedMessage);
     const { messageType, file } = failedMessage;
-    if (failedMessage?.isResendable) {
+    if (typeof failedMessage?.isResendable === 'boolean') {
```
Comment (liamcho) [inline src/modules/Channel/context/hooks/useResendMessageCallback.js:14]:
```suggestion
    if (typeof failedMessage?.isResendable === 'boolean' && failedMessage.isResendable) {
```
?

Comment (AhyoungRyu) [reply src/modules/Channel/context/hooks/useResendMessageCallback.js:14]: Ah yeah, actually the original logic has no problem. Probably I don't even need the fix in this file at least. So I'm checking with Uday again.

**Comment by AhyoungRyu** [comment]
The whole change in this file will not fix the linked SBIssue cause the SBIssue is related to group channel but at least, I fixed the incorrect logic in the open channel msg resend hook.

**Review by HoonBaek — APPROVED** [review]

**Pull request merged**